Clinical-Genomics/cgstats
# Support iseq

Support iseq

How to install:
1. trial and error

How to test:
1. cgstats add --machine iseq --unaligned Unaligned-Y30I10I10 /home/hiseq.clinical/iseq/demux//20190528_FS10000534_7_BPC29611-3324

Expected outcome:
- [x] new entry in database
- [x] flowcell name is BPC29611-3324
- [x] pos is 'A'
- [x] hiseqtype is 'iseq'
- [x] stuffs looking ok

This is a minor since we only add new func without changing the old

- [x] R @ingkebil 
- [x] T @emiliaol @patrikgrenfeldt 
- [x] D @ingkebil
